app/scripts/lib/setupSentry.js

```diff
@@ -1,12 +1,13 @@
 import * as Sentry from '@sentry/browser';
 import { createModuleLogger, createProjectLogger } from '@metamask/utils';
 import { logger } from '@sentry/utils';
-import { AllProperties } from '../../../shared/modules/object.utils';
+import browser from 'webextension-polyfill';
 import { isManifestV3 } from '../../../shared/modules/mv3.utils';
```

Comment:
This was the only line with a conflict in the most recent merge with develop. I just had to preserve this import